src/function/table/sqlite/CMakeLists.txt

```diff
@@ -1,8 +1,7 @@
-add_library_unity(duckdb_func_sqlite
-                  OBJECT
-                  pragma_collations.cpp
-                  pragma_table_info.cpp
-                  sqlite_master.cpp)
+add_definitions(-DDUCKDB_SOURCE_ID="\""${GIT_COMMIT_HASH}"\"")
```

Comment:
I struggled a bit with this one, this is coming from the `tools/sql_api_wrapper` files. I tried to get a reference to the variable defined there, but my `CMake` skills are still poor and I couldn't, so I ended up copy pasting.